pages/ae/3rdparties/3rdparties-installation.adoc

```diff
@@ -4,10 +4,212 @@
 :page-folder: ae/3rdparties
 :page-description: Gravitee Alert Engine - Third Parties - Installation
 :page-toc: true
-:page-keywords: Gravitee, API Platform, Alert, Alert Engine, documentation, manual, guide, reference, api
+:page-keywords: Gravitee, API Platform, Alert, Alert Engine, documentation, manual, guide, reference, api, third-party, 3rd-party
 :page-layout: ae
 
-AE works both within and outside the Gravitee.io ecosystem. You can plug in your own backends and benefit
-from all of the same features.
+AE works both within and outside the Gravitee.io ecosystem.
+You can plug in your own backends and benefit from all the same features.
 
-NOTE: This section will be updated soon.
+== Requirements
+
+You must have an Alert Engine available:
+
+* already deployed on your infrastructure
+* by running it with docker:
+** `docker run -t -v "${PWD}/licence.key:/opt/graviteeio-alert-engine/license/license.key:ro" -p 8072:8072 graviteeio/ae-engine:latest`
+** note that we use a licence.key file that you must have in the current directory (update the cli as you need)
+
+For running the javascript scripts below, we use `node version v18.7.0`.
+
+
+== Websocket connection
+
+You can send a triggers through websocket
```

Comment:
```suggestion
You can send a trigger through websocket.
```

Reply:
As we can send multiple triggers, I kept the `s` and removed the `a`. 😊